src/dataregistry/registrar/keyword.py

```diff
@@ -0,0 +1,216 @@
+import os
+from datetime import datetime
+from typing import Literal
+
+from sqlalchemy import select
+
+from dataregistry.db_basic import add_table_row
+from dataregistry.registrar.base_table_class import BaseTable
+from dataregistry.db_basic import DbConnection
+
+
+class KeywordTable(BaseTable):
+
+    """
+    The Keywords class is used to manage keywords in the data registry.
+    It provides methods to add, remove, and list keywords associated with datasets.
+
+    Parameters
+    ----------
+    db_connection : DbConnection object
+        Encompasses sqlalchemy engine, dialect (database backend)
+        and schema version
+    owner : str
+        To set the default owner for all registered keywords in this
+        instance.
+    """
+    def __init__(
+            self,
+            db_connection: DbConnection,
+            root_dir,
+            owner,
+            owner_type
+    ) -> None:
+        super().__init__(db_connection, root_dir, owner, owner_type)
+        self.which_table = "keyword"
+        self.entry_id = "keyword"
+
+    def add_keyword(
```

Comment:
I prefer to call this "create_keyword" so it won't be confused with "add_keywords" in the DatasetTable class.